tfk8s.go

```diff
@@ -88,6 +88,12 @@ func snakify(s string) string {
 	return strings.ToLower(re.ReplaceAllString(s, "_"))
 }
 
+// escape incidences of ${} with $${} to prevent Terraform trying to interpolate them
+func escapeShellVars(s string) string {
+	r := regexp.MustCompile(`(\${.*?})`)
+	return r.ReplaceAllString(s, `$$$1`)
```

Comment:
There is three `$` because `$1` uses the first group, then we need to escape the `$` we want to put in front of it 😅